contrib/go/src/python/pants/contrib/go/tasks/go_buildgen.py

```diff
@@ -231,9 +231,8 @@ def failed(self):
 
     def __str__(self):
       import_paths = ' '.join(sorted(self.import_paths))
-      rev = '' if self.local else ' {}'.format(self.rev or 'FLOATING')
-      return ('{build_file_path} ({import_paths}){rev}'
-              .format(build_file_path=self.build_file_path, import_paths=import_paths, rev=rev))
+      rev = '' if self.local else f" {(self.rev or 'FLOATING')}"
```

Comment:
I think the parentheses are unnecessary
```suggestion
      rev = '' if self.local else f" {self.rev or 'FLOATING'}"
```